### PR TITLE
Add equalWidthTrack wrapper

### PR DESCRIPTION
**Pull request by Eldwick**
In regards to #20 

Pass state and nodeWidths to direct children of ScrollTrack.

### CODE REVIEW

File: src/components/ScrollTrack/ScrollTrack.js [hunk 1]
```diff
@@ -56,6 +59,14 @@ class ScrollTrack extends Component {
     window.removeEventListener('resize', this.debouncdComputeSlideAttributes)
   }
 
+  get childrenWithTrackProps() {
+    const nodeWidths = this.getNodeWidths()
+
+    return React.Children.map(this.props.children, child => {
+      return React.cloneElement(child, { ...this.state, ...nodeWidths })
+    })
+  }
```
Comment (dcocchia) [inline src/components/ScrollTrack/ScrollTrack.js]: Should we namespace these? They're somewhat generically named within the component, and may collide with the children's own props.

Comment (Eldwick) [inline src/components/ScrollTrack/ScrollTrack.js]: 👍 good point

File: src/components/ScrollTrack/equalWidthTrack.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react'
```
Comment (dcocchia) [inline src/components/ScrollTrack/equalWidthTrack.js]: Can we add a comment block to the top of this file to describe it a bit more? When/why to use it, for example. We should add similar documentation in the ScrollTrack markdown file.

Comment (Eldwick) [inline src/components/ScrollTrack/equalWidthTrack.js]: Will do

**Pull request merged**